.huskyrc.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   hooks: {
     'pre-commit': 'yarn lint',
-    'pre-push': 'yarn test',
```

Comment:
Were the tests working before the changes introduced by this PR?

Reply:
No @netoax, I removed the old test code and this make this check fail. This is why I removed it! As you comment, maybe it's better to move this PR to draft until I add the appropriate test code. What do you think?

Reply:
I think if the tests weren't working before anyway, we can proceed with this PR.

Reply:
Yes, the old tests aren't passing :disappointed: